src/components/grocery-view/grocery-section/GrocerySection.js

```diff
@@ -1,23 +1,41 @@
 // import PropTypes from 'prop-types';
-// import React from 'react';
+import React from 'react';
 
-// // Components
-// import GroceryItem from './GroceryItem';
-// import CategoryRow from './CategoryRow';
+// Components
+import { Grid } from '@material-ui/core';
+import CategoryRow from './CategoryRow';
 
-// // Styles
+// Styles
 // import styles from '../styles/grocery-section.module.css';
 
-// const GrocerySection = () => (
-//     <div>
-//       GrocerySection
-//     </div>
-// );
+const GrocerySection = () => (
+  <div>
+    <h1>Groceries</h1>
+    <p>Here is some information about groceries</p>
+    <Grid
+      style={{ width: '100%' }}
+      container
+      direction="column"
+      alignContents="flex-end"
+      alignItems="stretch">
+      <Grid item>
+        <h2>Produce</h2>
+        <CategoryRow />
+      </Grid>
```

Comment:
I think it might be better to put all of this within the CategoryRow component, therefore we can autogenerate the rows based on the data we have. 
This would then have the future-potential of being generated from data (e.g. creating a vegan row). Should be a easy fix.
But eventually, we'll want to generate these rows from the data I think.